keras_cv/models/backbones/vgg19/vgg19_backbone_presets.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""VGG19 model preset configurations."""
+
+backbone_presets_no_weights = {
```

Comment:
This is an interesting corner case that @jbischof perhaps can weigh in on -- what do we do when we have a preset that is basically just a default for `include_rescaling` (VGG has a simpler API so presets are less meaningful).

Perhaps this should be a `VGGBackbone` and `vgg19` and `vgg16` should be presets?